src/core/api/languageTools.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import { callApi } from 'core/api';
+import type { ApiStateType } from 'core/reducers/api';
+
+
+export type LanguageToolsParams = {|
+  api: ApiStateType,
+|};
+
+export function languageTools({ api }: LanguageToolsParams) {
+  return callApi({
+    endpoint: 'addons/language-tools',
```

Comment:
The API docs suggest that this URL should have a trailing slash

Reply:
`callApi` adds the trailing slash.